src/web/app/pages-help/instructor-help-page/instructor-help-page.component.ts

```diff
@@ -30,6 +31,17 @@ export class InstructorHelpPageComponent implements OnInit {
     }
   }
 
+  /**
+   * Scrolls the screen to the section name.
+   */
+  scrollTo(section: string): void {
+    const query: any = document.querySelector(`#${section}`);
```

Comment:
Doing direct DOM manipulation is discouraged in Angular. Consider using [ViewChild](https://angular.io/api/core/ViewChild) to match the element in the HTML file and performing `scrollIntoView` on it